proposals/0007-live-edge.md

```diff
@@ -0,0 +1,218 @@
+- Seekable Live Edge = Advertised Live Edge - (`#EXT-X-SERVER-CONTROL:PART-HOLD-BACK` value)
+
+__Context__:
+
+> PART-HOLD-BACK
+> 
+> The value is a decimal-floating-point number of seconds that indicates the server-recommended minimum distance from the end of the Playlist at which clients should begin to play or to which they should seek when playing in Low-Latency Mode. Its value MUST be at least twice the Part Target Duration. Its value SHOULD be at least three times the Part Target Duration. If different Renditions have different Part Target Durations then PART-HOLD-BACK SHOULD be at least three times the maximum Part Target Duration.
+
+  \- From: https://datatracker.ietf.org/doc/html/draft-pantos-hls-rfc8216bis-12#section-4.4.3.8
+
+### Recommended computation of Live Edge - ISO/IEC 23009-1 (aka "MPEG-DASH")
+
+1. __"Standard Latency" Live__
+
+- Let Advertised Live Edge be the computed latest presentation time / duration available as derived from the `MPD` at time T, using `MPD@availabilityStartTime` for reference, and presuming an NTP time-synchronized client and server.
+  - NOTE: Since there are many ways to represent `AdaptationSet`s and their segments, describing the various ways of computing the Advertised Live Edge will be treated as out of scope for this proposal.
+- Seekable Live Edge = Advertised Live Edge - (`MPD@suggestedPresentationDelay` value ?? client player specific value ?? 0)
+
+__Context__:
+
+MPD@suggestedPresentationDelay: 
+> it specifies a fixed delay offset in time from the presentation time of each access unit that is suggested to be used for presentation of each access unit... When not specified, then no value is provided and the client is expected to choose a suitable value.
+
+  \- From: *§5.3.1.2 Table 3 - Semantics of MPD element*
+
+
+2. __Low Latency Live__
+
+- Let Reference Time be derived from the `ProducerReferenceTime` values which correlate to the presentation timeline and wallclock times 
+  - NOTE: Since there are many ways to represent `AdaptationSet`s and their segments, describing the various ways of computing these values will be treated as out of scope for this proposal.
+- Seekable Live Edge = Reference Time - (`ServiceDescription` -> `Latency@target` value)
+
+__Context__:
+
+> The service provider's preferred presentation latency in milliseconds compared to the producer reference time. Indicates a content provider's desire for the content to be presented as close to the indicated latency as is possible given the player's capabilities and observations.
+>
+> This attribute may express latency that is only achievable by low-latency players under favourable network conditions.
+
+  \- From: *Annex K, §K.3.2 Table K.1 - Service Latency* (See Also: *Table K.6 — Semantics of Latency element* from the same section)
+
+## Property: `liveEdgeStart` (to model Live Edge Window start)
```

Comment:
I think this section should come before the ` ## Property: Constraint on `HTMLMediaElement::seekable.end()` (to model Seekable Live Edge)` section as this is the important bit of this proposal.

I even wonder if that other section should move to a separate proposal or note.